src/System Application/App/Retention Policy/src/Retention Policy Setup/RetentionPolicySetupCard.Page.al

```diff
@@ -86,6 +86,16 @@ page 3901 "Retention Policy Setup Card"
                     ToolTip = 'Specifies the number of expired records.';
                     Editable = false;
                     StyleExpr = ExpiredRecordCountStyleTxt;
+                    Visible = Rec."Apply to all records";
+                }
+                field("Records To Delete"; ExpiredRecordCount)
+                {
+                    ApplicationArea = All;
+                    Caption = 'Records to delete';
+                    ToolTip = 'Specifies the number of expired records that will be deleted during the next retention policy run.';
```

Comment:
Specifies the number of expired records the retention policy will delete the next time it runs.